inspirehep/modules/theme/templates/inspirehep_theme/header.html

```diff
@@ -194,7 +194,8 @@
           <invenio-search-autocomplete-bar
             template="{{ url_for('static', filename='node_modules/inspirehep-search-js/dist/templates/searchbar.html') }}"
             placeholder="Search {{collection_name}}"
-            collection={{collection_name}}>
+            collection={{collection_name}}
+            allowAutocomplete="false">
```

Comment:
I've named the atrribute `allowAutocomplete` because just ` autocomplete` was generating errors.

Reply:
👍 we can name it ``allow-autocomplete``, angular automatically transform it to camel case ``allowAutocomplete`` inside the directive :)